app/javascript/packs/application.js.erb

```diff
@@ -0,0 +1,31 @@
+import "core-js/stable"
```

Comment:
Out of curiosity, what causes us to need this? I'm hopeful we can remove the import. Not a blocker.